src/System.Globalization/tests/project.lock.json

```diff
@@ -3,6 +3,17 @@
   "version": -9996,
   "targets": {
     "DNXCore,Version=v5.0": {
+      "System.AppContext/4.0.0-beta-23024": {
```

Comment:
The changes happen automatically when I build. I was not sure if they are needed. I can exclude them.